python/pacemaker/_cts/environment.py

```diff
@@ -240,28 +253,40 @@ def service_is_enabled(self, node, service):
             # explicitly "enabled" instead of the return code. For example it returns
             # 0 if the service is "static" or "indirect", but they don't really count
             # as "enabled".
-            (rc, _) = self._rsh(node, "systemctl is-enabled %s | grep enabled" % service)
+            (rc, _) = self._rsh(
+                node, "systemctl is-enabled %s | grep enabled" % service
+            )
             return rc == 0
 
         # SYS-V
         (rc, _) = self._rsh(node, "chkconfig --list | grep -e %s.*on" % service)
         return rc == 0
 
     def _detect_at_boot(self):
-        """ Detect if the cluster starts at boot """
+        """Detect if the cluster starts at boot"""
 
         if "at-boot" not in self.data:
-            self["at-boot"] = self.service_is_enabled(self._target, "corosync") \
-                              or self.service_is_enabled(self._target, "pacemaker")
+            self["at-boot"] = self.service_is_enabled(
+                self._target, "corosync"
+            ) or self.service_is_enabled(self._target, "pacemaker")
```

Comment:
Function candidate. OTOH service_is_enabled() is always called with self._target so we could drop that as an argument

Reply:
service_is_enabled is also called from CTStests.py, so we need the argument.